wcc-contentful-app/spec/helpers/wcc/contentful/app/section_helper_spec.rb

```diff
@@ -317,11 +363,17 @@
         ' [Watermark Community Church](http://www.watermark.org){: .button-medium .green}'\
         ' [Test](https://test.com).'
 
+      text_without_class_syntax =
+        +"Ministry developed by [Awaken](/awaken \"Awaken's Homepage\")"\
+        ' [Watermark Community Church](http://www.watermark.org)'\
+        ' [Test](https://test.com).'
+
       expect(text.include?('{: .button .white}')).to be true
 
-      helper.remove_markdown_href_class_syntax(raw_classes, text)
+      transformed_text = helper.remove_markdown_href_class_syntax(raw_classes, text)
 
-      expect(text.include?('{: .button .white}')).to be false
+      expect(transformed_text.include?('{: .button .white}')).to be false
```

Comment:
`expect(transformed_text).to_not include('{: .button .white}')` works also

Reply:
https://relishapp.com/rspec/rspec-expectations/v/3-8/docs/built-in-matchers/predicate-matchers

Reply:
Yeah, that's much nicer.